### PR TITLE
docs: document TabHeader props

### LINKED ISSUE #2562: Document props of `<TabHeader>`
**Which part of the documentation does it affect and which topic does it refer to?**

The tabular components (https://docs.commercetools.com/custom-applications/api-reference/commercetools-frontend-application-components) use a component `<TabHeader>` that is exported with the package as well.

However, it has been noted that it would help to explicitly document the props of this component.

**Describe what needs to be improved**

Document the `<TabHeader>` props.

### PR DESCRIPTION
**Pull request by kark**
#### Summary

The aim of this PR is to document `<TabHeader>` component props.

closes #2562

### CODE REVIEW

**Review by emmenko — APPROVED** [review]
Thanks! 👌

File: website/src/content/api-reference/commercetools-frontend-application-components.mdx [hunk 1]
```diff
@@ -759,6 +760,22 @@ const DetailPage = () => {
   This is a convenience proxy export to expose pre-defined Intl messages defined in the `@commercetools-frontend/i18n` package.
   The Intl messages can for instance be used for button labels.
 
+## TabHeader
+
+Tab Headers facilitate navigation within a tabular page.
+
+In a tabular page, tabs must be rendered using the `<TabHeader>` component via the `tabControls` prop. A `<TabHeader>` is rendered as a link and it assumes the "tab content" is controlled and rendered using `<Route>` components.
```
Comment (CarlosCortizasCT) [inline website/src/content/api-reference/commercetools-frontend-application-components.mdx:767]: Perhaps we could also explain that the component _reads_ the current URL path to determine its `active` state.

**Review by CarlosCortizasCT — APPROVED** [review]
Super helpful 👍 

I left a small suggestion to maybe consider.

File: website/src/content/api-reference/commercetools-frontend-application-components.mdx [hunk 2]
```diff
@@ -759,6 +760,23 @@ const DetailPage = () => {
...
+`<TabHeader>` tracks the route changes internally and a tab is active if the `to` prop matches the current route.
```
[the remark this answers is not in the document]
Comment (CarlosCortizasCT) [reply website/src/content/api-reference/commercetools-frontend-application-components.mdx:768]: It works for me.
Thanks 👍

**Review by CarlosCortizasCT — APPROVED** [review]
💯

**Review by tdeekens — APPROVED** [review]
Neat! /cc @ahmehri

**Comment by emmenko** [comment]
Waiting for #2608 to be merged first

**Review by ddouglasz — APPROVED** [review]

**Pull request merged**